chore: auth-wraper docs (#815)
cognitedata/cognitesdk-js: README.md

They show how to include the cognite SDK in various project setups.
The samples' [README.md](./samples/README.md) has instructions for running the samples.

## Authentication

> We offer two ways to get logged on our sdk, the first and most recommended is using [@cognite/auth-wrapper](./guides/auth-wrapper.md)
> and the second is using [Microsoft MSAL Library](./guides/auth-wrapper.md)

Reviewer: ```suggestion
We offer two ways to sign in to our SDK. The first and recommended approach is using [@cognite/auth-wrapper](./guides/auth-wrapper.md)
and the second is using [Microsoft MSAL Library](./guides/auth-wrapper.md)
```
